File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/ocp/excercise_00/HotdogIngredientsPackage.java
package es.emergya.labs.training.solid_java_1.ocp.excercise_00;

public class HotdogIngredientsPackage implements IngredientsInterface {

	private static String message = "Pan frying hothog with %.2fKg of cheese, %.2fKg of meat and %.2fKg of mayo";

	private Float cheeseKg;
	private Float meatKg;
	private Float mayoKg;

	public HotdogIngredientsPackage(Float cheeseKg, Float meatKg, Float mayoKg) {
		super();
		this.cheeseKg = cheeseKg;
		this.meatKg = meatKg;
		this.mayoKg = mayoKg;
	}

	public void makeFood(){
		panfry();
	}

	void panfry() {

		System.out.println(String.format(message, cheeseKg, meatKg, mayoKg));
	}
}

File: labs-training-master/SolidPythonResuelto/main/ocp/HotDogIngredientsPackage.py
from IngredientsPackageInterface import IngredientsPackageInterface

class HotdogIngredientsPackage(IngredientsPackageInterface) :
    def __init__(self, cheeseKg:str,  meatKg:str,  mayoKg:str) :
        self.cheeseKg = cheeseKg
        self.meatKg = meatKg
        self.mayoKg = mayoKg
        self.message:str = "Pan frying hothog with {0}Kg of cheese, {1}Kg of meat and {2}Kg of mayo".format(str(cheeseKg),str(meatKg),str(mayoKg))

    def panfry(self) -> None:
        print(self.message)

    def makeFood(self) -> None:
        self.panfry()